fuzz/fuzz_targets/against_bitvec.rs

```diff
@@ -75,11 +75,14 @@ impl ReadBitmapOp {
                 let vec_iter = b.to_vec();
                 assert!(vec_iter.into_iter().eq(v.iter_ones().map(|i| i as u32)));
             }
-            ReadBitmapOp::GetSerializedSizeInBytes => {
-                b.get_serialized_size_in_bytes();
+            ReadBitmapOp::GetPortableSerializedSizeInBytes => {
+                b.get_serialized_size_in_bytes::<Portable>();
```

Comment:
Need to import Portable/Native/Frozen

Reply:
Thanks -- fixed in [a455945](https://github.com/saulius/croaring-rs/pull/106/commits/a4559452e9148ce58f22cde918b82294bc87f929). I wonder why `make fuzz` did not fail for me locally 🤔

Reply:
Looks like `make fuzz` just runs the fuzz_ops fuzzer